docs/src/components/HeroSection/HeroSection.js

```diff
@@ -29,24 +30,28 @@ const getClassName = cssModules(STYLES);
 
 type Props = {
   className: string,
-  imageUrl: string,
+  imageUrl?: string,
   heading: string,
   subHeading: ?string,
 };
 
 const HeroSection = (props: Props) => {
   const { className, imageUrl, heading, subHeading } = props;
-  const style = {
+
+  const style = imageUrl && {
     backgroundImage: `url(${imageUrl})`,
   };
+
   const classNames = [getClassName('bpkdocs-hero-section'), className];
   return (
     <div className={classNames.join(' ')} style={style}>
       <Heading
         level="h1"
         className={getClassName('bpkdocs-hero-section__heading')}
       >
-        {heading}
+        <BpkText textStyle="xxl" bold>
```

Comment:
Is this going to be applicable to all the new pages? We might be able to just pass the textStyle and bold into the Heading as i believe it uses BpkText under the hood?

Reply:
Yes is for all pages, but if you look at the design they all have this style so I don't see it as a problem